package.json

```diff
@@ -27,9 +27,9 @@
   },
   "homepage": "https://github.com/montymxb/parse-server-test#readme",
   "dependencies": {
-    "forever": "0.15.3",
+    "forever": "1.0.0",
     "mongodb-runner": "4.7.1",
-    "parse-server": "github:parse-community/parse-server#master",
-    "winston": "2.4.3"
+    "parse-server": "3.3.0",
```

Comment:
Can you revert this to point to the latest branch?

Reply:
why?  then if the master branch is broken, the php repo will be broken too? seems better to test against the current release?

Reply:
If a new feature is added we would have to wait for a server release (which would take a while) before we can update the SDK's. It's the same for the JS SDK.

Reply:
k.  done.